src/Home/HomeElrond.js

```diff
@@ -1,23 +1,22 @@
-import React, { useState, useEffect } from "react";
-import { Box, Stack } from "@chakra-ui/layout";
-import { 
-  Button, Badge, Spacer, Modal, ModalOverlay, ModalContent, ModalHeader, ModalBody, ModalCloseButton, ModalFooter, 
-  Text, HStack, Heading, Wrap, Image, WrapItem, Spinner, useToast, useDisclosure, useBreakpointValue } from "@chakra-ui/react";
+import React, { useState, useEffect } from 'react';
+import { Box, Stack } from '@chakra-ui/layout';
+import { Button, Badge, Spacer, Modal, ModalOverlay, ModalContent, ModalHeader, ModalBody, ModalCloseButton, ModalFooter, Text, HStack, Heading, Wrap, Image, WrapItem, Spinner, useToast, useDisclosure, useBreakpointValue } from '@chakra-ui/react';
 import { WarningTwoIcon } from '@chakra-ui/icons';
-import moment from "moment";
-import { progInfoMeta, uxConfig, debugui } from "libs/util";
-import { CHAIN_TOKEN_SYMBOL, CLAIM_TYPES, MENU, SUPPORTED_CHAINS } from "libs/util";
-import imgProgGaPa from "img/prog-gaming.jpg";
-import imgProgRhc from "img/prog-rhc.png";
-import imgProgWfh from "img/prog-wfh.png";
-import myNFMe from "img/my-nfme.png";
-import ClaimModalElrond from "ClaimModel/ClaimModalElrond";
-import { useUser } from "store/UserContext";
-import { useChainMeta } from "store/ChainMetaContext";
-import ChainSupportedComponent from "UtilComps/ChainSupportedComponent";
-import { FaucetContract } from "Elrond/faucet";
-import { ClaimsContract } from "Elrond/claims";
-import { useGetAccountInfo, useGetPendingTransactions, useGetLoginInfo } from "@elrondnetwork/dapp-core";
+import moment from 'moment';
+import { progInfoMeta, uxConfig, debugui } from 'libs/util';
+import { CHAIN_TOKEN_SYMBOL, CLAIM_TYPES, MENU, SUPPORTED_CHAINS } from 'libs/util';
+import imgProgGaPa from 'img/prog-gaming.jpg';
+import imgProgRhc from 'img/prog-rhc.png';
+import imgProgWfh from 'img/prog-wfh.png';
+import myNFMe from 'img/my-nfme.png';
+import ClaimModalElrond from 'ClaimModel/ClaimModalElrond';
+import { useUser } from 'store/UserContext';
+import { useChainMeta } from 'store/ChainMetaContext';
+import ChainSupportedComponent from 'UtilComps/ChainSupportedComponent';
+import { FaucetContract } from 'Elrond/faucet';
+import { ClaimsContract } from 'Elrond/claims';
+import { useGetAccountInfo, useGetPendingTransactions, useGetLoginInfo } from '@elrondnetwork/dapp-core';
+import { TokenPayment } from '@elrondnetwork/erdjs/out';
```

Comment:
@damienen what is this TokenPayment for?

Reply:
You're right, that should not have been there.

I solved it.

It remained as an unused dependency from erdjs after I tried using that class to solve the bug (but I ended using some other solution)